### DISCUSSION TITLE
Unable to start mcserver

### DISCUSSION THREAD

**Discussion by abruncke** [GameServerManagers/LinuxGSM#3460] [Q&A] [answered]
I've been running lgsm (mcserver) for a while, and recently wanted to start a new map / version.  I updated lgsm to 21.2.3, installed a new forge server jar file (forge-1.16.5-36.1.0.jar), updated game config to point to the new jar file, but any attempt to start gives me "[FAIL] Starting mcserver: Unable to start SERVER_NAME".

I can run the actual command line that I am using manually, and the server starts just fine.  I have not made any other lgsm / mcserver config file or directory changes.

I've tried to debug and check things to see what has changed, but am a bit overwhelmed / lost as to where to check first.  Any help is appreciated.

**Comment by Claiyc** [comment, marked as answer]
You need to use java 8 if you want to run forge. Did you make sure that this is the case?

**Reply by abruncke** [reply]
Yup, been running java 8 and using forge for a year now.  This just started happening with an update to lgsm, and switching to a new forge version.  Again, the actual server will run fine, I just can't get it to run from ./mcserver start.

**Reply by abruncke** [reply]
Solved it.  Seems like a change that came through with my latest update of lgsm was some changes to how the minecraft executable is built with config files, names, etc.  Just changed things around to match the new structure and got it working.
